src/Engine/Engine.php

```diff
@@ -133,14 +134,9 @@ public function addTest(
         $filter = $filter ? $filter : $this->filter;
         $chooser = $chooser ? $chooser : $this->chooser;
 
-        if (null === $filter) {
-            throw new \RuntimeException('There must be at least one filter. In the Engine or in the TestBag');
-        }
-
-        if (null === $chooser) {
-            throw new \RuntimeException('There must be at least one chooser. In the Engine or in the TestBag');
-        }
-
+        Assert::notNull($filter, 'There must be at least one filter. In the Engine or in the TestBag');
```

Comment:
mmm, is this splitting the sentence in two intentional?
I'd go for:
"There must be at least one filter in the Engine or in the TestBag."